### ISSUE TITLE
Bug: Sql import indexes are not parsed properly

### ISSUE THREAD

**Issue by luckcolors** [vuerd/vuerd#238]
When importing an SQL table containing indexes those are not parsed properly into index properties but are instead left as keys in the table.

To reproduce:

SQL Code:
` ` ` SQL
CREATE TABLE 'users' (
  'id' bigint unsigned NOT NULL AUTO_INCREMENT,
  'name' varchar(30) NOT NULL,
  'email' varchar(30) NOT NULL,
  PRIMARY KEY ('id'),
  UNIQUE KEY 'users_email_unique' ('email'),
  KEY 'test_name_index' ('name'),
);
` ` ` 

Result:

![immagine](https://user-images.githubusercontent.com/3228598/132374886-855fb842-8d6b-4616-8156-6bf95d70124b.png)

Additionally the index section of the table properties is not populated.

![immagine](https://user-images.githubusercontent.com/3228598/132375077-a70bf332-ac65-4579-8832-6e9ddd2301e0.png)

**Comment by luckcolors** [comment]
Additional info on KEY index: `[see](https://stackoverflow.com/questions/1401572/whats-the-difference-between-using-index-vs-key-in-mysql)`.

**Comment by dineug** [comment]
vcode extension v0.9.7 has been reflected.

**Comment by luckcolors** [comment]
Seems to be partially fixed both indexes are not present as keys in the table, but only the "KEY" index is added to the indexes of the table. The unique index is missing.

Try to run my example SQL.

**Comment by dineug** [comment]
Automatically generated indexes are not explicitly managed as index data.

**Comment by luckcolors** [comment]
Okay but an   UNIQUE index isn't automatically generated or rather the user still needs to add it.
So i would say since you're saving for every entity the indices it has, this index is then lost on import.

**Comment by dineug** [comment]
Are you looking for these?
<img width="516" alt="스크린샷 2021-09-09 오전 2 57 56" src="https://user-images.githubusercontent.com/45829489/132560585-e2795c51-1da8-4523-823a-1abdac424173.png">

**Comment by luckcolors** [comment]
Nono that's not what i mean.
I'm saying that on the Database->Indexes page only the name index is listed, the unique email index is not there.

![immagine](https://user-images.githubusercontent.com/3228598/132562276-0283295f-2055-43d0-9164-962816e99c43.png)

**Comment by luckcolors** [comment]
Yeah, if i add the view change i can see it aswell. But it is still not present in the table properties tab.

**Comment by dineug** [comment]
Doesn't UNIQUE KEY generate indexes automatically?

**Comment by luckcolors** [comment]
It does, but i think it makes sense for it to be in the list of indexes.
Also it wether a key is unique or not is not taken into account in the generated SQL atm it seems.

![immagine](https://user-images.githubusercontent.com/3228598/132563133-fb0b1b3b-b89a-4c4d-b15b-ea48f2dc6918.png)

I personally think all indexes should be listed in the table properties indexes section of the app, it can be confusing to the user to see some indexes only when you change the app view.

Honestly i doesn't really matter where you define the indexes: wether it be immediately in the CREATE TABLE statement or in a following ADD INDEX statement.

**Comment by dineug** [comment]
I checked that it was missing from the simple table SQL DDL.

<img width="1260" alt="스크린샷 2021-09-09 오전 3 36 14" src="https://user-images.githubusercontent.com/45829489/132566079-e6d1cd96-366c-4b8e-ae99-7104510fcd75.png">

PRIMARY KEY, UNIQUE KEY can't be edited.
I think it needs to be done in a way that only shows the list.

**Comment by luckcolors** [comment]
Okay no, I wasn't talking about the Primary key. That's fine the way it is.

I'm talking about the custom UNIQUE index i'm creating on email.

According to [this](https://dev.mysql.com/doc/refman/8.0/en/create-index.html)
You can actually add the unique index just fine later with the CREATE INDEX statement (it automatically does the alter table for you).

I think i understand what you where saying earlier.
I suppose it's up to you wether you want to list all indexes in the Table Properties Indexes.
I don't think it's going to be problematic, might even be more user friendly, you could aswell list the primary key index there aswell.

Now how you actually generate the table could change based on how every index is defined:

 - The PRIMARY INDEX ofcourse needs to happend in the CREATE TABLE.
 - 
 - UNIQUE index with only 1 key can be rapresented with the UNIQUE keyword in the table definition query.

 -  Other more complex index could be added either as a KEY statement or with extra CREATE INDEX statements.
 - 
Again i don't think it matters that much as all of the SQL it is generating is going to be used to create a DATABASE from scratch.
I just think it can be confusing not listing just UNIQUE immediate indexes in the Table Properties, as you will only be able to see them by changing the table view.

**Comment by luckcolors** [comment]
For example the SQL query i was tesing with can be rapresented as:

`SQL
CREATE TABLE 'users' (
  'id' bigint unsigned NOT NULL AUTO_INCREMENT,
  'name' varchar(30) NOT NULL,
  'email' varchar(30) NOT NULL UNIQUE,
  PRIMARY KEY ('id'),
  KEY 'user_name_index' ('name')
);
`
Or as:

`SQL
CREATE TABLE 'users' (
  'id' bigint unsigned NOT NULL AUTO_INCREMENT,
  'name' varchar(30) NOT NULL,
  'email' varchar(30) NOT NULL,
  PRIMARY KEY ('id'),
);

CREATE UNIQUE INDEX users_email_index
ON users(email ASC);

CREATE  INDEX users_name_index
ON users(name ASC);  
`
